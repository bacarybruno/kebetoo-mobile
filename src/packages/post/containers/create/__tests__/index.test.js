import configureStore from 'redux-mock-store'

import setupTest from 'Kebetoo/src/config/jest-setup'

import CreatePost from '../index'

const mockStore = configureStore()
const store = mockStore()

const givenCreatePost = setupTest(CreatePost)({
  navigation: {
    navigate: jest.fn(),
    setOptions: jest.fn(),
  },
  store,
})

it('renders CreatePost', () => {
  const { wrapper } = givenCreatePost()
  expect(wrapper.toJSON()).toMatchSnapshot()
})
