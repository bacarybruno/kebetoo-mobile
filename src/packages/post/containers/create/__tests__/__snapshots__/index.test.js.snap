// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CreatePost 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "paddingBottom": 24,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "marginHorizontal": 16,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#F2F2F2",
            "borderColor": "#E8E8E8",
            "borderRadius": 15,
            "borderWidth": 1,
            "minHeight": 48,
            "paddingLeft": 16,
          },
          Object {
            "backgroundColor": "#F2F2F2",
            "marginTop": 8,
            "paddingRight": 16,
          },
        ]
      }
    >
      <TextInput
        allowFontScaling={true}
        autoFocus={true}
        blurOnSubmit={false}
        defaultValue=""
        maxLength={180}
        multiline={true}
        numberOfLines={8}
        onChangeText={[Function]}
        placeholder="What's on your mind?"
        placeholderTextColor="#ACACAC"
        rejectResponderTermination={true}
        returnKeyType="done"
        style={
          Array [
            Object {
              "fontSize": 16,
            },
            Object {
              "textAlignVertical": "top",
            },
          ]
        }
        underlineColorAndroid="transparent"
        value={null}
      />
    </View>
    <View
      style={
        Object {
          "flex": 1,
          "justifyContent": "flex-end",
        }
      }
    />
    <View
      style={
        Object {
          "alignItems": "flex-end",
          "flexDirection": "row",
          "justifyContent": "space-between",
          "marginTop": 16,
        }
      }
    >
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "row",
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
            }
          }
        >
          <View
            accessible={true}
            focusable={true}
            hitSlop={
              Object {
                "bottom": 20,
                "left": 20,
                "right": 20,
                "top": 20,
              }
            }
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#F2F2F2",
                "borderRadius": 20,
                "height": 40,
                "justifyContent": "center",
                "marginRight": 16,
                "opacity": 1,
                "transform": Array [
                  Object {
                    "scale": 1,
                  },
                ],
                "width": 40,
              }
            }
          >
            <Text
              allowFontScaling={false}
              style={
                Object {
                  "color": "#0B1B77",
                  "fontFamily": "icomoon",
                  "fontSize": 18.018018018018015,
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                }
              }
              text="kebeticon-camera"
            />
          </View>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
            }
          }
        >
          <View
            accessible={true}
            focusable={true}
            hitSlop={
              Object {
                "bottom": 20,
                "left": 20,
                "right": 20,
                "top": 20,
              }
            }
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#F2F2F2",
                "borderRadius": 20,
                "height": 40,
                "justifyContent": "center",
                "marginRight": 16,
                "opacity": 1,
                "transform": Array [
                  Object {
                    "scale": 1,
                  },
                ],
                "width": 40,
              }
            }
          >
            <Text
              allowFontScaling={false}
              style={
                Object {
                  "color": "#0B1B77",
                  "fontFamily": "icomoon",
                  "fontSize": 18.018018018018015,
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                }
              }
              text="kebeticon-photo"
            />
          </View>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
            }
          }
        >
          <View
            accessible={true}
            focusable={true}
            hitSlop={
              Object {
                "bottom": 20,
                "left": 20,
                "right": 20,
                "top": 20,
              }
            }
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#F2F2F2",
                "borderRadius": 20,
                "height": 40,
                "justifyContent": "center",
                "marginRight": 16,
                "opacity": 1,
                "transform": Array [
                  Object {
                    "scale": 1,
                  },
                ],
                "width": 40,
              }
            }
          >
            <Text
              allowFontScaling={false}
              style={
                Object {
                  "color": "#0B1B77",
                  "fontFamily": "icomoon",
                  "fontSize": 18.018018018018015,
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                }
              }
              text="kebeticon-more-h"
            />
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
          }
        }
      >
        <View
          accessible={true}
          focusable={false}
          hitSlop={
            Object {
              "bottom": 20,
              "left": 20,
              "right": 20,
              "top": 20,
            }
          }
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#F2F2F2",
              "borderRadius": 20,
              "height": 40,
              "justifyContent": "center",
              "marginRight": 16,
              "opacity": 1,
              "transform": Array [
                Object {
                  "scale": 1,
                },
              ],
              "width": 40,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Object {
                "color": "#0B1B77",
                "fontFamily": "icomoon",
                "fontSize": 18.018018018018015,
                "fontStyle": "normal",
                "fontWeight": "normal",
              }
            }
            text="kebeticon-microphone"
          />
        </View>
      </View>
    </View>
  </View>
</View>
`;
